service/job.go

```diff
@@ -141,3 +145,38 @@ func (s *CaptionsService) CreateJob(r *http.Request) (int, interface{}, error) {
 
 	return http.StatusCreated, job, nil
 }
+
+// DownloadCaption downloads a caption in the specified format
+func (s *CaptionsService) DownloadCaption(w http.ResponseWriter, r *http.Request) {
+	id := web.Vars(r)["id"]
+
+	requestLogger := s.logger.WithFields(log.Fields{
+		"Handler": "DownloadCaption",
+		"Method":  r.Method,
+		"URI":     r.RequestURI,
+	})
+	param := downloadParam{}
+
+	defer r.Body.Close()
+
+	data, err := ioutil.ReadAll(r.Body)
+	if err != nil {
+		requestLogger.WithError(err).Error("Could not read download request body: ")
+		w.WriteHeader(http.StatusBadRequest)
+	}
+
+	err = json.Unmarshal(data, &param)
+	if err != nil {
+		requestLogger.WithError(err).Error("Could not unmarshal download request body")
+		w.WriteHeader(http.StatusBadRequest)
+	}
+
+	captionFile, err := s.client.DownloadCaption(id, param.CaptionFormat)
+	if err != nil {
+		w.WriteHeader(http.StatusNotFound)
+	}
+
+	w.Header().Set("Content-Type", fmt.Sprintf("text/%s", param.CaptionFormat))
```

Comment:
Can you append ``; charset="utf-8"`` to the Content-Type just to avoid encoding issues?